bindata/etcd/pod.yaml

```diff
@@ -20,47 +20,93 @@ spec:
         #!/bin/sh
         set -euo pipefail
 
-        echo "NODE__NAME=NODE_NAME"
-        echo "NODE__ENVVAR_NAME=NODE_ENVVAR_NAME"
-        echo "NODE_NODE__ENVVAR_NAME_ETCD_DNS_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}"
-        echo "NODE_NODE__ENVVAR_NAME_ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME}"
-        echo "NODE_NODE__ENVVAR_NAME_IP=${NODE_NODE_ENVVAR_NAME_IP}"
-
         sleep 24h
 
-        # exec etcd \
-        #        --initial-advertise-peer-urls=https://${NODE_NODE_ENVVAR_NAME_IP}:2380 \
-        #        --cert-file=/etc/ssl/etcd/system:etcd-server:${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}.crt \
-        #        --key-file=/etc/ssl/etcd/system:etcd-server:${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}.key \
-        #        --trusted-ca-file=/etc/ssl/etcd/ca.crt \
-        #        --client-cert-auth=true \
-        #        --peer-cert-file=/etc/ssl/etcd/system:etcd-peer:${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}.crt \
-        #        --peer-key-file=/etc/ssl/etcd/system:etcd-peer:${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}.key \
-        #        --peer-trusted-ca-file=/etc/ssl/etcd/ca.crt \
-        #        --peer-client-cert-auth=true \
-        #        --advertise-client-urls=https://${NODE_NODE_ENVVAR_NAME_IP}:2379 \
-        #        --listen-client-urls=https://${LISTEN_ON_ALL_IPS}:2379 \
-        #        --listen-peer-urls=https://${LISTEN_ON_ALL_IPS}:2380 \
-        #        --listen-metrics-urls=https://${LISTEN_ON_ALL_IPS}:9978 \
+        exit 0
+
+        # add logic here to confirm that we are part of the etcd members (the controller added us).
+        # this is probably a golang command that tries to confirm for two minutes before exiting
+        # and prints nothing except for the ETCD_INITIAL_CLUSTER
```

Comment:
can we put this logic in an init container to the Pod so we can have access to CEO commands?